### PR TITLE
Updated readme with essential content

### PR DESCRIPTION
**Pull request by Millyy**
:D

### CODE REVIEW

**Comment by Millyy** [comment]
🍡

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# git101
\ No newline at end of file
+# git101
+
+Celia is cool
```
Comment (Toxicable) [inline README.md:3]: Not sure about this one, might be able to replace it with a slightly lower Big Oh Algorithm

Comment (Millyy) [reply README.md:3]: Yeah sorry, I couldn't find an algorithm with a lower complexity than O(n^5). It's the sacrifice you have to make for such an important addition!

Comment (Toxicable) [reply README.md:3]: Well in that case I guess it's unavoidable

**Comment by Toxicable** [comment]
LGTM

**Pull request merged**